TriforkSwiftExtensions/TriforkSwiftExtensions/Numeric/NumericExtension.swift

```diff
@@ -18,9 +17,9 @@ public extension Double {
     public func string(withMaxDigits maxDigits: Int) -> String? {
         let formatter = NumberFormatter()
         formatter.maximumSignificantDigits = maxDigits
-        return formatter.string(from: NSNumber(value: self))
+        return formatter.string(from: self as! NSNumber)
```

Comment:
This should always be safe as Numeric == NSNumber. 
I can safely cast and return nil if you prefer that @tkctrifork

Reply:
I see why it is safe, since the `NSNumber` constructor interface supports all types of `Numeric` implementations, but this code might be unsafe if Apple decides to change this. The `NSNumber` doesn't support the `Numeric` protocol, which is why I think we should unwrap this safely before passing it to the constructor.